deploy/scripts/configure.sh

```diff
@@ -19,10 +19,17 @@ VNET_RESOURCE_GROUP=${VNET_RESOURCE_GROUP:-"${ENV_CODE}-vnet-rg"}
 DATA_RESOURCE_GROUP=${DATA_RESOURCE_GROUP:-"${ENV_CODE}-data-rg"}
 PROCESSING_RESOURCE_GROUP=${PROCESSING_RESOURCE_GROUP:-"${ENV_CODE}-processing-rg"}
 
+APIM_SERVICE_NAME=$(az apim list --resource-group $PROCESSING_RESOURCE_GROUP --query "[0].value" -o tsv)
+
+APIM_STAC_LOADBALANCER_URL=$(az apim api list --resource-group $PROCESSING_RESOURCE_GROUP \
+    --service-name $APIM_SERVICE_NAME --filter-display-name fast-stac-api)
+
 SUBSCRIPTION=$(az account show --query id -o tsv)
 AZURE_APP_INSIGHTS=$(az resource list -g $MONITORING_RESOURCE_GROUP --resource-type "Microsoft.Insights/components" \
     --query "[?tags.environment && tags.environment == '$ENV_NAME'].name" -o tsv)
 
+LOADBALANCER_IP=$(echo "$x" | awk -F/ '{print $3}' | awk -F: '{print $1}')
```

Comment:
`$x` seems like maybe something that was left from testing?

Reply:
Yes, was supposed to be $APIM_STAC_LOADBALANCER_URL, not $x. Updated it. I'm planning to run one single end-to-end test at the end.